maps/us-healthcare-system-capacity/js/main.js

```diff
@@ -330,10 +347,25 @@ function setLegend(colors, minMaxValues) {
 }
 
 function setCirclePaintStyle(layerName) {
-  var colors = indicators[indicator].colors;
-  var minMaxValues = getMinMax();
+  var colorsMinMax = indicators[indicator].colors,
+      radii = indicators[indicator].radii,
+      breaksValues = getBreaks(),
+      palette = interpolate(colorsMinMax),
+      colors = _.map([...Array(breaksValues.length).keys()], function(i) {
+        return palette(i / breaksValues.length);
+      }),
+      radiiZ1 = _.map([...Array(breaksValues.length).keys()], function(i) {
+        return ((i / breaksValues.length) * 19) + 1;
+      }),
+      radiiZ2 = _.map([...Array(breaksValues.length).keys()], function(i) {
+        return ((i / breaksValues.length) * 5) + 5;
+      });
```

Comment:
Seeing now that I left the radii unimplmented; they looked OK once I changed the radius targets for bed occupancy. This is TODO to be implemented later.